pagopa/pn-frontend
# Feat/pn 2409 add ot script privacy and tos

## Short description
This PR adds the retrieval of the privacy policy from OneTrust CDN and  the script that parses the privacy policy json, to pa-webapp, personafisica-login and personafisica-webapp.
This PR does not include the script for personagiuridica webapp

## List of changes proposed in this pull request
- Adds useEffect to retrieve privacy policy json
- Adds scripts that parses privacy policy json
- Removes privacy policy old content

## How to test
- Add these to your .env.local variables:
-- pn-personafisica-login > REACT_APP_ONE_TRUST_PORTAL_CDN = https://privacyportalde-cdn.onetrust.com/77f17844-04c3-4969-a11d-462ee77acbe1/privacy-notices/draft/133242e7-ad9a-45e5-a417-b7a50c746899.json
-- pn-pa-webapp > REACT_APP_ONE_TRUST_PORTAL_CDN = https://privacyportalde-cdn.onetrust.com/77f17844-04c3-4969-a11d-462ee77acbe1/privacy-notices/draft/4824a110-316e-42fd-b492-8e2b0513db70.json
-- pn-personafisica-webapp > REACT_APP_ONE_TRUST_PORTAL_CDN = https://privacyportalde-cdn.onetrust.com/77f17844-04c3-4969-a11d-462ee77acbe1/privacy-notices/draft/365c84c5-9329-4ec5-89f5-e53572eda132.json
- Check that privacy policy loads the privacy policy draft correctly in every app
